javav2/example_code/s3/README.md

```diff
@@ -17,7 +17,7 @@ Amazon S3 is cloud object storage with industry-leading scalability, data availa
 
 ### Single action
 
-You will find these examples that use the **S3Client** object: 
+You will find these examples that use the **S3Client** class: 
 
 - [Aborting a multipart upload to an Amazon S3 bucket](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/s3/src/main/java/com/example/s3/AbortMultipartUpload.java) (AbortMultipartUpload command)
```

Comment:
For these links, it's OK to say "S3 bucket" instead of "Amazon S3 bucket".

I believe we are also moving away from using gerunds in these link titles, so "Aborting" would be "Abort," "Copying" would be "Copy," and so forth.